index.d.ts

```diff
@@ -40,6 +40,30 @@ export function useStore<
   ? Pick<StoreValue<SomeStore>, Key>
   : StoreValue<SomeStore>
 
+type Listener<SomeStore extends Store> = (
+  value: StoreValue<SomeStore>,
+  changed?: SomeStore extends MapStore ? StoreValue<SomeStore> : never
```

Comment:
Does this parameter only have a value if the store is `map` store?

And what is its value if the store value is a primitive?

Reply:
> Does this parameter only have a value if the store is map store?

Yeap

> And what is its value if the store value is a primitive?

In this case you do not have `store.setKey` and we have no easy way to detect changes

Reply:
Well I was asking because of this: https://github.com/nanostores/react/pull/2/files#r744618456